meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio_%.bbappend

```diff
@@ -40,3 +40,14 @@ do_deploy_append_mx8m() {
         ln -sf u-boot-spl.bin-${MACHINE} u-boot-spl.bin-${MACHINE}-
     fi
 }
+
+DEPENDS_append_sota_freedom-u540 = " opensbi"
+
+do_compile_prepend_sota_freedom-u540() {
+    export OPENSBI=${DEPLOY_DIR_IMAGE}/fw_dynamic.bin
+}
+
+UBOOT_BUILD_DEP_sota_freedom-u540 = "opensbi:do_deploy"
```

Comment:
We can probably make this the default for lmp and lmp-base (on freadom-u540), no? It might require a new wic file that is not sota specific, but it is easy to add (guess similar to the one from meta-sifive).

Reply:
Yep.  I've made this change apply to any of our builds and uploaded a new wic to support lmp-base.